include/net/ieee802154_radio.h

```diff
@@ -162,6 +162,11 @@ enum ieee802154_config_type {
 
 	/** Configure a radio reception slot */
 	IEEE802154_CONFIG_RX_SLOT,
+
+	/** Enable/disable or update Enhanced-ACK Based Probing in radio
+	 *  for a specific Initiator.
+	 */
+	IEEE802154_CONFIG_ENH_ACK_PROBING,
```

Comment:
It's a bit unclear to me, OpenThread enables probing, and what then? Will this trigger some callbacks from the radio side?

Reply:
From this moment the device should include the configured link metrics values in every Enh-ACK generated for this neighbor.

Reply:
Ah, so that's how it works, thanks for clarifying.